tokens/src/lib.rs

```diff
@@ -10,14 +13,23 @@ use srml_support::{decl_error, decl_event, decl_module, decl_storage, ensure, Pa
 // #3295 https://github.com/paritytech/substrate/issues/3295
 use srml_system::{self as system, ensure_signed};
 
-use traits::MultiCurrency;
+use traits::{arithmetic, arithmetic::Signed, MultiCurrency, MultiCurrencyExtended};
 
 mod mock;
 mod tests;
 
 pub trait Trait: srml_system::Trait {
 	type Event: From<Event<Self>> + Into<<Self as srml_system::Trait>::Event>;
 	type Balance: Parameter + Member + SimpleArithmetic + Default + Copy + MaybeSerializeDeserialize;
+	type Amount: arithmetic::Signed
```

Comment:
you `use arithmetic::Signed` and still `arithmetic::Signed` here?

Reply:
Aha, forgot to change it after adding `Signed` to support `abs()` and `is_positive()`.